clientv3/concurrency/election.go

```diff
@@ -39,17 +39,14 @@ type Election struct {
 }
 
 // NewElection returns a new election on a given key prefix.
-func NewElection(client *v3.Client, pfx string) *Election {
+func NewElection(client *ClientWithSession, pfx string) *Election {
```

Comment:
@heyitsanthony Is this what you want? In your example, election only takes session as arg. But it is not enough since we still need to create keys by using the client for example.